Modules and packages/Import module 2/task.md

```diff
@@ -0,0 +1,15 @@
+## Import module 2
+
+You can import not only functions: you can import classes or even other modules. It is customary but not required to place all 
+import statements at the beginning of a module.
+
+You can find out more about modules in Python by reading [this section](https://docs.python.org/3/tutorial/modules.html) of The Python Tutorial.
+
+### Task
+In the code editor, import the module `calculator` and create an instance of the class `Calculator` (`calc`). 
```

Comment:
In the code editor, import the `calculator` module and create an instance of the `Calculator` class (`calc`).